crates/subspace-archiving/src/utils.rs

```diff
@@ -0,0 +1,7 @@
+pub(crate) fn slice_to_arrays<S: AsRef<[u8]> + ?Sized>(slice: &S) -> Vec<[u8; 2]> {
+    slice
+        .as_ref()
+        .chunks_exact(2)
```

Comment:
Nice to have a comment that briefly explains the magic number here. If I remember correctly, it's actually from the library reed_solomon_erasure, right?

Reply:
Yes, it is from `reed_solomon_erasure`. Updated implementation.